server/router.go

```diff
@@ -1,8 +1,21 @@
 package server
 
-import "github.com/gin-gonic/gin"
+import (
+	"github.com/gin-gonic/gin"
+	"os"
+	"strings"
+)
 
 func setupRoutes(router *gin.Engine) {
+	var dir, _ = os.Getwd()
+	var path string
+	if strings.Contains(dir,"server"){
+		path = "../template/*"
+	}else{
+		path = "template/*"
+	}
```

Comment:
A comment of what this logic is doing would be helpful for understanding the code